### PR TITLE
Generate Geodatabase sample

### PR DESCRIPTION
**Pull request by nCastle1**
Sample is implemented for all platforms.

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT can you please take a look at this sample when you get a chance?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/GenerateGeodatabase/GenerateGeodatabase.xaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.GenerateGeodatabase.GenerateGeodatabase"
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/GenerateGeodatabase/GenerateGeodatabase.xaml]: I don't like this UI for the WPF version (it works great for the others). I think maybe a border in the upper-left with the button and status bar (and maybe a descriptive label) would be better.

**Comment by ThadT** [comment]
Reviewed and tested. Thanks @nCastle1

**Comment by nCastle1** [comment]
All platforms tested and verified post-merge.

**Pull request merged**